### PR TITLE
Add more details to Telerivet error message

### PR DESCRIPTION
**Pull request by millerdev**
@dannyroberts

### CODE REVIEW

File: corehq/messaging/smsbackends/telerivet/models.py [hunk 1]
```diff
@@ -99,8 +99,16 @@ def send(self, msg, *args, **kwargs):
...
+            try:
+                data = response.text
+            except Exception:
+                data = repr(response.content)
             raise TelerivetException(
-                "Received HTTP response status code %s from backend %s" % (response.status_code, self.pk)
+                "Received HTTP response status code %s (%s) from backend %s" % (
+                    response.status_code,
+                    self.pk,
+                    data,
```
Comment (kaapstorm) [inline corehq/messaging/smsbackends/telerivet/models.py]: Did you intend to move `self.pk` from after "backend" to inside the parens? Or did you intend `data` to be inside the parens? ... Might be clearer if you switch to `.format()`.

Comment (millerdev) [reply corehq/messaging/smsbackends/telerivet/models.py]: good catch. I will fix

Comment (millerdev) [reply corehq/messaging/smsbackends/telerivet/models.py]: https://github.com/dimagi/commcare-hq/pull/24001/commits/93010dfb742013c17b3749f89b3617286209c3cf

Comment (millerdev) [reply corehq/messaging/smsbackends/telerivet/models.py]: Eh, was reading too fast and missed your recommendation to `.format()` until after I had already pushed the commit. Not going to bother unless you object.

Comment (kaapstorm) [reply corehq/messaging/smsbackends/telerivet/models.py]: Yeah, definitely don't bother. (Also, it's 2020 now: The days of `.format()` are over.)

**Review by kaapstorm — APPROVED** [review]

**Pull request merged**